.github/auto-label.yaml

```diff
@@ -0,0 +1,26 @@
+---
+path:
+  pullrequest: true
+  paths:
+    doc: 'documentation'
+    .github: 'workflows'
+    terraform: 'workflows'
+    etc:
+      kayobe:
+        enviromnents:
+          ci-aio: 'workflows'
+          ci-builder: 'workflows'
+          ci-multinode: 'workflows'
+        trivy: 'workflows'
+        ansible: 'ansible'
+        kolla:
+          config:
+            grafana: "monitoring"
+            prometheus: "monitoring"
+            fluentd: "monitoring"
+
+staleness:
+  pullrequest: true
```

Comment:
We can change this in the follow-up PR when we add more tweaks, but imo the defaults for staleness are too long. (60/120 days) https://github.com/googleapis/repo-automation-bots/tree/main/packages/auto-label#staleness-label